alerting/src/main/kotlin/org/opensearch/alerting/action/GetRemoteIndexesResponse.kt

```diff
@@ -42,7 +42,7 @@ class GetRemoteIndexesResponse : ActionResponse, ToXContentObject {
 
     data class ClusterIndexes(
         val clusterName: String,
-        val clusterHealth: ClusterHealthStatus,
+        val clusterHealth: ClusterHealthStatus?,
```

Comment:
is making `clusterHealth` optional intentional? if yes, can we add a exists check? if no, plz remove

Reply:
It is intentional for the the health status to be optional, as a user may have index privileges on a cluster, but not cluster health API privileges. Will add exist checks.